desktop/src/com/mygdx/game/client/di/SetterBindingsModule.java

```diff
@@ -0,0 +1,34 @@
+package com.mygdx.game.client.di;
+
+import com.mygdx.game.client.ecs.entityfactory.ChooseableSetter;
+import com.mygdx.game.client.ecs.entityfactory.DesktopCoordinateSetter;
+import com.mygdx.game.client.ecs.entityfactory.ModelInstanceCompSetter;
+import com.mygdx.game.client.ecs.entityfactory.NavigationDirectionSetter;
+import com.mygdx.game.client.ecs.entityfactory.TextureCompSetter;
+import com.mygdx.game.client_core.ecs.entityfactory.Setter;
+import dagger.Binds;
+import dagger.Module;
+import dagger.multibindings.IntoSet;
+
+@Module
+public interface SetterBindingsModule {
```

Comment:
No niby spoko, ale nie widzę żadnych zalet tego rozwiązania, nie jest to raczej ładniejsze ani czytelniejsze

Reply:
no mi też się średnio podoba xD
ale podoba mi się, że można poćwiczyć jakieś bardziej zaawansowane funkcje daggera